### PR TITLE
New recipe libvips/8.14.2

### PR DESCRIPTION
**Pull request by SpaceIm**
closes https://github.com/conan-io/conan-center-index/issues/969

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libvips/all/test_package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
+    test_type = "explicit"
+
+    def layout(self):
+        cmake_layout(self)
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def build(self):
```
Comment (kamilszulc-tomtom) [inline recipes/libvips/all/test_package/conanfile.py:18]: why don't we use meson?
def build(self):
        meson = Meson(self)
        meson.configure()
        meson.build()

**Comment by SpaceIm** [comment]
```
WARN: libtiff/4.4.0: requirement libwebp/1.2.4 overridden by libvips/8.14.1 to libwebp/1.3.0 
ERROR: Missing binary: glib/2.75.3:5257b1b4173c70cac90c89a38a10ecb6e777b84c
glib/2.75.3: WARN: Can't find a 'glib/2.75.3' package for the specified settings, options and dependencies:
- Settings: arch=x86_64, build_type=Release, compiler=Visual Studio, compiler.runtime=MT, compiler.version=16, os=Windows
- Options: shared=True, with_pcre=True, bzip2:build_executable=True, bzip2:shared=False, libffi:shared=False, libgettext:shared=False, libgettext:threads=windows, libiconv:shared=False, pcre2:build_pcre2_16=True, pcre2:build_pcre2_32=True, pcre2:build_pcre2_8=True, pcre2:build_pcre2grep=True, pcre2:grep_support_callout_fork=True, pcre2:shared=False, pcre2:support_jit=False, pcre2:with_bzip2=True, pcre2:with_zlib=True, zlib:shared=False
- Dependencies: zlib/1.2.13, libffi/3.4.3, pcre2/10.40, libgettext/0.21
- Requirements: libffi/3.Y.Z, libgettext/0.21, pcre2/10.Y.Z, zlib/1.Y.Z
- Package ID: 5257b1b4173c70cac90c89a38a10ecb6e777b84c

ERROR: Missing prebuilt package for 'glib/2.75.3'
Use 'conan search glib/2.75.3@ --table=table.html -r=remote' and open the table.html file to see available packages
Or try to build locally from sources with '--build=glib'
```

mhhh I guess I have to drop MT runtime since glib shared MT package is not generated by c3i

**Comment by SpaceIm** [comment]
```
[34/485] Compiling C object libvips/foreign/libforeign.a.p/nsgifload.c.obj
FAILED: libvips/foreign/libforeign.a.p/nsgifload.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glibb5189374a6143/p/include" "-IC:/Users/spaceim/.conan2/p/glibb5189374a6143/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glibb5189374a6143/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/pcre2e08a977b0538e/p/include" "-IC:/Users/spaceim/.conan2/p/zlib448b8d3597eb3/p/include" "-IC:/Users/spaceim/.conan2/p/bzip2047c1b64797cc/p/include" "-IC:/Users/spaceim/.conan2/p/libge96846d6ce83d9/p/include" "-IC:/Users/spaceim/.conan2/p/libic43f068348dd12/p/include" "-IC:/Users/spaceim/.conan2/p/libff312af4a45a008/p/include" "-IC:/Users/spaceim/.conan2/p/expatddf07e68bf2df/p/include" "-IC:/Users/spaceim/.conan2/p/fftw31acda33f9c75/p/include" "-IC:/Users/spaceim/.conan2/p/libjpfc2a9cde73a8e/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2f604e8a43791/p/include" "-IC:/Users/spaceim/.conan2/p/libwe93ccdb38a8fed/p/include" "-IC:/Users/spaceim/.conan2/p/libti589469df6c3fc/p/include" "-IC:/Users/spaceim/.conan2/p/libdec0e816bdd1bf3/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut7e7f5d1ae7e17/p/include" "-IC:/Users/spaceim/.conan2/p/jbige8511ccc8662e/p/include" "-IC:/Users/spaceim/.conan2/p/zstd865c25c9aebea/p/include" "-IC:/Users/spaceim/.conan2/p/lcmsfca33f5d0a132/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DCMS_DLL" "-DLIBDEFLATE_DLL" "-D_JBIGDLL_" "/Fdlibvips\foreign\libforeign.a.p\nsgifload.c.pdb" /Folibvips/foreign/libforeign.a.p/nsgifload.c.obj "/c" ../src/libvips/foreign/nsgifload.c
../src/libvips/foreign/nsgifload.c(329): warning C4244: 'function': conversion from 'double' to 'int', possible loss of data
../src/libvips/foreign/nsgifload.c(491): error C2036: 'nsgif_bitmap_t *': unknown size
[39/485] Compiling C object libvips/foreign/libforeign.a.p/ppmload.c.obj
FAILED: libvips/foreign/libforeign.a.p/ppmload.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glibb5189374a6143/p/include" "-IC:/Users/spaceim/.conan2/p/glibb5189374a6143/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glibb5189374a6143/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/pcre2e08a977b0538e/p/include" "-IC:/Users/spaceim/.conan2/p/zlib448b8d3597eb3/p/include" "-IC:/Users/spaceim/.conan2/p/bzip2047c1b64797cc/p/include" "-IC:/Users/spaceim/.conan2/p/libge96846d6ce83d9/p/include" "-IC:/Users/spaceim/.conan2/p/libic43f068348dd12/p/include" "-IC:/Users/spaceim/.conan2/p/libff312af4a45a008/p/include" "-IC:/Users/spaceim/.conan2/p/expatddf07e68bf2df/p/include" "-IC:/Users/spaceim/.conan2/p/fftw31acda33f9c75/p/include" "-IC:/Users/spaceim/.conan2/p/libjpfc2a9cde73a8e/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2f604e8a43791/p/include" "-IC:/Users/spaceim/.conan2/p/libwe93ccdb38a8fed/p/include" "-IC:/Users/spaceim/.conan2/p/libti589469df6c3fc/p/include" "-IC:/Users/spaceim/.conan2/p/libdec0e816bdd1bf3/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut7e7f5d1ae7e17/p/include" "-IC:/Users/spaceim/.conan2/p/jbige8511ccc8662e/p/include" "-IC:/Users/spaceim/.conan2/p/zstd865c25c9aebea/p/include" "-IC:/Users/spaceim/.conan2/p/lcmsfca33f5d0a132/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DCMS_DLL" "-DLIBDEFLATE_DLL" "-D_JBIGDLL_" "/Fdlibvips\foreign\libforeign.a.p\ppmload.c.pdb" /Folibvips/foreign/libforeign.a.p/ppmload.c.obj "/c" ../src/libvips/foreign/ppmload.c
../src/libvips/foreign/ppmload.c(487): error C2036: 'const void *': unknown size
```

To fix for Visual Studio build.

**Comment by SpaceIm** [comment]
```
ERROR: libvips/8.14.1: Error in source() method, line 207
	get(self, **self.conan_data["sources"][self.version], strip_root=True)
	AuthenticationException: ﻿<?xml version="1.0" encoding="utf-8"?><Error><Code>AuthenticationFailed</Code><Message>Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.
RequestId:887f8ec4-701e-004c-2733-55b255000000
```

hum

**Comment by SpaceIm** [comment]
```
ERROR: Missing binary: fftw/3.3.9:6c63e27ff3b8f63bd848ab7b966d115b8db9dc16
ERROR: Missing binary: lcms/2.14:abe5e2b04ea92ce2ee91bc9834317dbe66628206
ERROR: Missing binary: libspng/0.7.3:e08b13a511b71813d560f3f1f26e06304f68e714

fftw/3.3.9: WARN: Can't find a 'fftw/3.3.9' package binary '6c63e27ff3b8f63bd848ab7b966d115b8db9dc16' for the configuration:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.version=11
os=Linux
[options]
openmp=False
precision=double
shared=True
simd=False
threads=False

ERROR: Missing prebuilt package for 'fftw/3.3.9', 'lcms/2.14', 'libspng/0.7.3'
Check the available packages using 'conan list fftw/3.3.9:* -r=remote'
or try to build locally from sources using the '--build=fftw/3.3.9 --build=lcms/2.14 --build=libspng/0.7.3' argument
```

@danimtb @uilianries another bunch of v2 packages to generate here.

**Comment by SpaceIm** [comment]
Local test of libvips static with Visual Studio: it fails to open libintl.h (from a glib public header), which means that libgettext is a public dependency of glib, but is private currently in glib recipe.

```
[8/462] Compiling C object libvips/foreign/libforeign.a.p/analyzeload.c.obj
FAILED: libvips/foreign/libforeign.a.p/analyzeload.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/zlibb13707403285d/p/include" "-IC:/Users/spaceim/.conan2/p/expat0f108fbf7f42d/p/include" "-IC:/Users/spaceim/.conan2/p/fftw1f56ccb4f6cb0/p/include" "-IC:/Users/spaceim/.conan2/p/libjp3d943c61374de/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2bb50fb464fff/p/include" "-IC:/Users/spaceim/.conan2/p/libweb6a00492a1112/p/include" "-IC:/Users/spaceim/.conan2/p/libticc75abcf017d1/p/include" "-IC:/Users/spaceim/.conan2/p/libded01d0631c40da/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut9128c9d0b92ad/p/include" "-IC:/Users/spaceim/.conan2/p/jbigb205bf71d5a4b/p/include" "-IC:/Users/spaceim/.conan2/p/zstd32c716410c623/p/include" "-IC:/Users/spaceim/.conan2/p/lcms1e5e21cf67953/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DLZMA_API_STATIC" "-DSPNG_STATIC" "-DLIBJPEG_STATIC" "-DXML_STATIC" "/Fdlibvips\foreign\libforeign.a.p\analyzeload.c.pdb" /Folibvips/foreign/libforeign.a.p/analyzeload.c.obj "/c" ../src/libvips/foreign/analyzeload.c
C:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0\glib/gi18n-lib.h(25): fatal error C1083: Cannot open include file: 'libintl.h': No such file or directory
[9/462] Compiling C object libvips/foreign/libforeign.a.p/analyze2vips.c.obj
FAILED: libvips/foreign/libforeign.a.p/analyze2vips.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/zlibb13707403285d/p/include" "-IC:/Users/spaceim/.conan2/p/expat0f108fbf7f42d/p/include" "-IC:/Users/spaceim/.conan2/p/fftw1f56ccb4f6cb0/p/include" "-IC:/Users/spaceim/.conan2/p/libjp3d943c61374de/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2bb50fb464fff/p/include" "-IC:/Users/spaceim/.conan2/p/libweb6a00492a1112/p/include" "-IC:/Users/spaceim/.conan2/p/libticc75abcf017d1/p/include" "-IC:/Users/spaceim/.conan2/p/libded01d0631c40da/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut9128c9d0b92ad/p/include" "-IC:/Users/spaceim/.conan2/p/jbigb205bf71d5a4b/p/include" "-IC:/Users/spaceim/.conan2/p/zstd32c716410c623/p/include" "-IC:/Users/spaceim/.conan2/p/lcms1e5e21cf67953/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DLZMA_API_STATIC" "-DSPNG_STATIC" "-DLIBJPEG_STATIC" "-DXML_STATIC" "/Fdlibvips\foreign\libforeign.a.p\analyze2vips.c.pdb" /Folibvips/foreign/libforeign.a.p/analyze2vips.c.obj "/c" ../src/libvips/foreign/analyze2vips.c
C:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0\glib/gi18n-lib.h(25): fatal error C1083: Cannot open include file: 'libintl.h': No such file or directory
[10/462] Compiling C object libvips/foreign/libforeign.a.p/csvload.c.obj
FAILED: libvips/foreign/libforeign.a.p/csvload.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/zlibb13707403285d/p/include" "-IC:/Users/spaceim/.conan2/p/expat0f108fbf7f42d/p/include" "-IC:/Users/spaceim/.conan2/p/fftw1f56ccb4f6cb0/p/include" "-IC:/Users/spaceim/.conan2/p/libjp3d943c61374de/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2bb50fb464fff/p/include" "-IC:/Users/spaceim/.conan2/p/libweb6a00492a1112/p/include" "-IC:/Users/spaceim/.conan2/p/libticc75abcf017d1/p/include" "-IC:/Users/spaceim/.conan2/p/libded01d0631c40da/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut9128c9d0b92ad/p/include" "-IC:/Users/spaceim/.conan2/p/jbigb205bf71d5a4b/p/include" "-IC:/Users/spaceim/.conan2/p/zstd32c716410c623/p/include" "-IC:/Users/spaceim/.conan2/p/lcms1e5e21cf67953/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DLZMA_API_STATIC" "-DSPNG_STATIC" "-DLIBJPEG_STATIC" "-DXML_STATIC" "/Fdlibvips\foreign\libforeign.a.p\csvload.c.pdb" /Folibvips/foreign/libforeign.a.p/csvload.c.obj "/c" ../src/libvips/foreign/csvload.c
C:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0\glib/gi18n-lib.h(25): fatal error C1083: Cannot open include file: 'libintl.h': No such file or directory
[11/462] Compiling C object libvips/foreign/libforeign.a.p/csvsave.c.obj
FAILED: libvips/foreign/libforeign.a.p/csvsave.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/zlibb13707403285d/p/include" "-IC:/Users/spaceim/.conan2/p/expat0f108fbf7f42d/p/include" "-IC:/Users/spaceim/.conan2/p/fftw1f56ccb4f6cb0/p/include" "-IC:/Users/spaceim/.conan2/p/libjp3d943c61374de/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2bb50fb464fff/p/include" "-IC:/Users/spaceim/.conan2/p/libweb6a00492a1112/p/include" "-IC:/Users/spaceim/.conan2/p/libticc75abcf017d1/p/include" "-IC:/Users/spaceim/.conan2/p/libded01d0631c40da/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut9128c9d0b92ad/p/include" "-IC:/Users/spaceim/.conan2/p/jbigb205bf71d5a4b/p/include" "-IC:/Users/spaceim/.conan2/p/zstd32c716410c623/p/include" "-IC:/Users/spaceim/.conan2/p/lcms1e5e21cf67953/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DLZMA_API_STATIC" "-DSPNG_STATIC" "-DLIBJPEG_STATIC" "-DXML_STATIC" "/Fdlibvips\foreign\libforeign.a.p\csvsave.c.pdb" /Folibvips/foreign/libforeign.a.p/csvsave.c.obj "/c" ../src/libvips/foreign/csvsave.c
C:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0\glib/gi18n-lib.h(25): fatal error C1083: Cannot open include file: 'libintl.h': No such file or directory
[12/462] Compiling C object libvips/foreign/libforeign.a.p/cgifsave.c.obj
FAILED: libvips/foreign/libforeign.a.p/cgifsave.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/zlibb13707403285d/p/include" "-IC:/Users/spaceim/.conan2/p/expat0f108fbf7f42d/p/include" "-IC:/Users/spaceim/.conan2/p/fftw1f56ccb4f6cb0/p/include" "-IC:/Users/spaceim/.conan2/p/libjp3d943c61374de/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2bb50fb464fff/p/include" "-IC:/Users/spaceim/.conan2/p/libweb6a00492a1112/p/include" "-IC:/Users/spaceim/.conan2/p/libticc75abcf017d1/p/include" "-IC:/Users/spaceim/.conan2/p/libded01d0631c40da/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut9128c9d0b92ad/p/include" "-IC:/Users/spaceim/.conan2/p/jbigb205bf71d5a4b/p/include" "-IC:/Users/spaceim/.conan2/p/zstd32c716410c623/p/include" "-IC:/Users/spaceim/.conan2/p/lcms1e5e21cf67953/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DLZMA_API_STATIC" "-DSPNG_STATIC" "-DLIBJPEG_STATIC" "-DXML_STATIC" "/Fdlibvips\foreign\libforeign.a.p\cgifsave.c.pdb" /Folibvips/foreign/libforeign.a.p/cgifsave.c.obj "/c" ../src/libvips/foreign/cgifsave.c
C:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0\glib/gi18n-lib.h(25): fatal error C1083: Cannot open include file: 'libintl.h': No such file or directory
[13/462] Compiling C object libvips/foreign/libforeign.a.p/dzsave.c.obj
FAILED: libvips/foreign/libforeign.a.p/dzsave.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/zlibb13707403285d/p/include" "-IC:/Users/spaceim/.conan2/p/expat0f108fbf7f42d/p/include" "-IC:/Users/spaceim/.conan2/p/fftw1f56ccb4f6cb0/p/include" "-IC:/Users/spaceim/.conan2/p/libjp3d943c61374de/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2bb50fb464fff/p/include" "-IC:/Users/spaceim/.conan2/p/libweb6a00492a1112/p/include" "-IC:/Users/spaceim/.conan2/p/libticc75abcf017d1/p/include" "-IC:/Users/spaceim/.conan2/p/libded01d0631c40da/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut9128c9d0b92ad/p/include" "-IC:/Users/spaceim/.conan2/p/jbigb205bf71d5a4b/p/include" "-IC:/Users/spaceim/.conan2/p/zstd32c716410c623/p/include" "-IC:/Users/spaceim/.conan2/p/lcms1e5e21cf67953/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DLZMA_API_STATIC" "-DSPNG_STATIC" "-DLIBJPEG_STATIC" "-DXML_STATIC" "/Fdlibvips\foreign\libforeign.a.p\dzsave.c.pdb" /Folibvips/foreign/libforeign.a.p/dzsave.c.obj "/c" ../src/libvips/foreign/dzsave.c
C:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0\glib/gi18n-lib.h(25): fatal error C1083: Cannot open include file: 'libintl.h': No such file or directory
[14/462] Compiling C object libvips/foreign/libforeign.a.p/exif.c.obj
FAILED: libvips/foreign/libforeign.a.p/exif.c.obj
"cl" "-Ilibvips\foreign\libforeign.a.p" "-Ilibvips\foreign" "-I..\src\libvips\foreign" "-I." "-I..\src" "-Ilibvips\include" "-I..\src\libvips\include" "-Ilibvips\include\vips" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0" "-IC:/Users/spaceim/.conan2/p/glib66192a2feef50/p/lib/glib-2.0/include" "-IC:/Users/spaceim/.conan2/p/zlibb13707403285d/p/include" "-IC:/Users/spaceim/.conan2/p/expat0f108fbf7f42d/p/include" "-IC:/Users/spaceim/.conan2/p/fftw1f56ccb4f6cb0/p/include" "-IC:/Users/spaceim/.conan2/p/libjp3d943c61374de/p/include" "-IC:/Users/spaceim/.conan2/p/libsp2bb50fb464fff/p/include" "-IC:/Users/spaceim/.conan2/p/libweb6a00492a1112/p/include" "-IC:/Users/spaceim/.conan2/p/libticc75abcf017d1/p/include" "-IC:/Users/spaceim/.conan2/p/libded01d0631c40da/p/include" "-IC:/Users/spaceim/.conan2/p/xz_ut9128c9d0b92ad/p/include" "-IC:/Users/spaceim/.conan2/p/jbigb205bf71d5a4b/p/include" "-IC:/Users/spaceim/.conan2/p/zstd32c716410c623/p/include" "-IC:/Users/spaceim/.conan2/p/lcms1e5e21cf67953/p/include" "-DNDEBUG" "/MD" "/nologo" "/showIncludes" "/utf-8" "/W2" "/O2" "/Gw" "-DG_DISABLE_CAST_CHECKS" "-DG_DISABLE_CHECKS" "-DG_DISABLE_ASSERT" "-DHAVE_CONFIG_H=1" "-DLZMA_API_STATIC" "-DSPNG_STATIC" "-DLIBJPEG_STATIC" "-DXML_STATIC" "/Fdlibvips\foreign\libforeign.a.p\exif.c.pdb" /Folibvips/foreign/libforeign.a.p/exif.c.obj "/c" ../src/libvips/foreign/exif.c
C:/Users/spaceim/.conan2/p/glib66192a2feef50/p/include/glib-2.0\glib/gi18n-lib.h(25): fatal error C1083: Cannot open include file: 'libintl.h': No such file or directory
[15/462] Compiling C object libvips/foreign/libforeign.a.p/cairo.c.obj
ninja: build stopped: subcommand failed.
```

**Comment by SpaceIm** [comment]
v2 pipeline needs https://github.com/conan-io/conan-center-index/pull/16578, https://github.com/conan-io/conan-center-index/pull/16579 & https://github.com/conan-io/conan-center-index/pull/16580

**Comment by SpaceIm** [comment]
https://github.com/libvips/libvips/releases/tag/v8.14.2 has been released, and these patches for Visual Studio are not required anymore.

**Review by jwillikers — DISMISSED** [review]

File: recipes/libvips/all/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+        if not self.options.cpp:
+            self.settings.rm_safe("compiler.cppstd")
+            self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("expat/2.5.0")
+        self.requires("glib/2.76.0", transitive_headers=True, transitive_libs=True, run=can_run(self))
```
Comment (prince-chrismc) [inline recipes/libvips/all/conanfile.py:120]: 😄 Please comment why the libs are transitive 🙏

Comment (SpaceIm) [reply recipes/libvips/all/conanfile.py:120]: glib headers are included in most public headers of libvips, and several glib functions are called in these headers, so not sure what kind of comment you want that would be more useful than a self-explanatory `transitive_headers=True, transitive_libs=True`.

**Comment by SpaceIm** [comment]
```
ERROR: Missing prebuilt package for 'glib/2.76.1'
```

I don't understand why these packages are missing since there was 
https://github.com/conan-io/conan-center-index/pull/16663

@RubenRBS could you generate these packages please?

**Comment by SpaceIm** [comment]
I don't know whether these missing packages are generated, let's try...

**Review by jwillikers — APPROVED** [review]

**Pull request merged**